### PR TITLE
fix: [FFM-3324]: Re-enable pipeline step tab

### PR DESCRIPTION
**Pull request by laura-mannus**
##### Summary:

- removed code pertaining to `showAdvancedTab` in `src/modules/70-pipeline/components/PipelineStudio/StepCommands/StepCommands.tsx`

- updated associated tests to make sure that the advanced tab actually appears on the feature flags stage and module.

##### Jira Links:

https://harness.atlassian.net/browse/FFM-3324

##### Screenshots:

image

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Review by Kira-Jennings — DISMISSED** [review]

File: src/modules/70-pipeline/components/PipelineStudio/StepCommands/__tests__/StepCommands.test.tsx [hunk 1]
```diff
@@ -96,7 +96,7 @@ describe('<StepCommands /> tests', () => {
     expect(container).toMatchSnapshot()
   })
 
-  test('should hide advanced tab for feature flags module', async () => {
+  test('should not hide advanced tab for feature flags module', async () => {
```
Comment (knagurski) [inline src/modules/70-pipeline/components/PipelineStudio/StepCommands/__tests__/StepCommands.test.tsx:99]: You can drop both of these tests. We don't need to single FF out anymore

File: src/modules/70-pipeline/components/PipelineStudio/StepCommands/__tests__/StepCommands.test.tsx [hunk 2]
```diff
@@ -154,7 +154,7 @@ describe('<StepCommands /> tests', () => {
       </TestWrapper>
     )
 
-    expect(queryByText('advancedTitle')).not.toBeInTheDocument()
+    expect(queryByText('advancedTitle')).toBeInTheDocument()
   })
 
   test('should show advanced tab for module and stage which are not feature flag', async () => {
```
Comment (knagurski) [inline src/modules/70-pipeline/components/PipelineStudio/StepCommands/__tests__/StepCommands.test.tsx:160]: And this one might be void as well.

File: src/modules/70-pipeline/components/PipelineStudio/StepCommands/__tests__/StepCommands.test.tsx (hunk 1 above)
Comment (laura-mannus) [reply src/modules/70-pipeline/components/PipelineStudio/StepCommands/__tests__/StepCommands.test.tsx]: Removed them :)

File: src/modules/70-pipeline/components/PipelineStudio/StepCommands/__tests__/StepCommands.test.tsx (hunk 2 above)
Comment (laura-mannus) [reply src/modules/70-pipeline/components/PipelineStudio/StepCommands/__tests__/StepCommands.test.tsx]: Removed it

**Review by Kira-Jennings — APPROVED** [review]

**Comment by laura-mannus** [comment]
retrigger jest

**Review by knagurski — APPROVED** [review]
Don't ya just love ripping stuff out?

**Review by vkbansal — APPROVED** [review]

**Pull request merged**